environment-model/environment-model_shacl.ttl

```diff
@@ -1,136 +1,159 @@
+    sh:targetClass environment-model:Format .
+
+environment-model:ProjectShape a sh:NodeShape ;
+    sh:property [ skos:example "7.7" ;
             sh:datatype xsd:string ;
-            sh:description "tool for the creation of the data"@en ;
-            sh:maxCount "1"^^xsd:unsignedInt ;
-            sh:message "Validation of creation_source failed!"@en ;
-            sh:minCount "1"^^xsd:unsignedInt ;
-            sh:name "creation_source"@en ;
-            sh:path environment-model:environment-model_project_creation_source ],
-        [ skos:example "SDK 4.25" ;
+            sh:description "Tool for the creation of the data"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of creationVersion failed!"@en ;
+            sh:name "creationVersion"@en ;
+            sh:order 4 ;
+            sh:path environment-model:creationVersion ],
+        [ skos:example "Trian3DBuilder" ;
+            sh:datatype xsd:string ;
+            sh:description "Tool for the creation of the data"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of creationSource failed!"@en ;
+            sh:name "creationSource"@en ;
+            sh:order 3 ;
+            sh:path environment-model:creationSource ],
+        [ skos:example "Unreal" ;
+            sh:datatype xsd:string ;
+            sh:description "Name of the graphics engine"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of softwareName failed!"@en ;
+            sh:name "softwareName"@en ;
+            sh:order 0 ;
+            sh:path environment-model:softwareName ],
+        [ skos:example "Epic Games" ;
             sh:datatype xsd:string ;
-            sh:description "nan"@en ;
-            sh:maxCount "1"^^xsd:unsignedInt ;
+            sh:description "Name of software vendor"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of softwareVendor failed!"@en ;
+            sh:name "softwareVendor"@en ;
+            sh:order 1 ;
+            sh:path environment-model:softwareVendor ],
+        [ skos:example "4.27" ;
+            sh:datatype xsd:string ;
+            sh:description "Version of graphics engine"@en ;
+            sh:maxCount 1 ;
             sh:message "Validation of version failed!"@en ;
-            sh:minCount "1"^^xsd:unsignedInt ;
             sh:name "version"@en ;
-            sh:path environment-model:environment-model_data_version ],
-        [ skos:example "Epic" ;
-            sh:datatype xsd:string ;
-            sh:description "name of software vendor"@en ;
-            sh:maxCount "1"^^xsd:unsignedInt ;
-            sh:message "Validation of software_vendor failed!"@en ;
-            sh:minCount "1"^^xsd:unsignedInt ;
-            sh:name "software_vendor"@en ;
-            sh:path environment-model:environment-model_project_software_vendor ],
-        [ skos:example "PBR" ;
+            sh:order 2 ;
+            sh:path environment-model:version ] ;
```

Comment:
name clash? environment-model:version already defined.

Reply:
change other property to softwareVersion